migrator/migrations/m_175_to_m_176_create_notification_schedule_table/migration.go

```diff
@@ -0,0 +1,34 @@
+package m175tom176
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	frozenSchema "github.com/stackrox/rox/migrator/migrations/m_175_to_m_176_create_notification_schedule_table/schema"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"gorm.io/gorm"
+)
+
+var (
+	startSeqNum = 175
+	migration   = types.Migration{
+		StartingSeqNum: startSeqNum,
+		VersionAfter:   &storage.Version{SeqNum: int32(startSeqNum + 1)},
+		Run: func(database *types.Databases) error {
+			return createNotificationScheduleTable(database.PostgresDB, database.GormDB)
+		},
+	}
+)
+
+func init() {
+	migrations.MustRegisterMigration(migration)
+}
+
+func createNotificationScheduleTable(_ *postgres.DB, gormDB *gorm.DB) error {
+	ctx := context.Background()
+	pgutils.CreateTableFromModel(ctx, gormDB, frozenSchema.CreateTableNotificationSchedulesStmt)
```

Comment:
Is the sole purpose of this migration to create the table?  No data moved into it, simply create it?

Reply:
Correct. The associated store is a singleton store at the moment (single-item table). It will be populated on first use.

Reply:
Technically based on current implementation a migration is not necessary.  In `migrator/main.go` after the migrations have executed around line 204 we make a call to `pkgSchema.ApplyAllSchemas(context.Background(), gormDB)`.  That would apply such a table.  That said, I'm not opposed to having the migration because honestly I'm not sure I like having that call to apply what's left after the migrations.  I see pros and cons to that.